libtransact/src/execution/executor/internal.rs

```diff
@@ -354,6 +381,11 @@ impl ExecutorThread {
                         }
                     }
                 }
+
+                Self::shutdown_fanout_threads(&fanout_threads);
```

Comment:
This looks like a separate change; can you explain this in the commit message?

Reply:
Moved to its own commit.